app/assets/v2/js/pages/bounty_detail/web3_modal.js

```diff
@@ -2,16 +2,34 @@ const payWithWeb3 = (fulfillment_id, fulfiller_address,  vm, modal) => {
   
   const amount = vm.fulfillment_context.amount;
   const token_name = vm.bounty.token_name;
-  
-  web3.eth.sendTransaction({
-    to: fulfiller_address,
-    from: selectedAccount,
-    value: web3.utils.toWei(String(amount)),
-    gasPrice: web3.utils.toHex(5 * Math.pow(10, 9)),
-    gas: web3.utils.toHex(318730),
-    gasLimit: web3.utils.toHex(318730)
-  },
-  function(error, result) {
+
+  if (token_name == 'ETH') {
+    web3.eth.sendTransaction(
+      {
+        to: fulfiller_address,
+        from: selectedAccount,
+        value: web3.utils.toWei(String(amount)),
+        gasPrice: web3.utils.toHex(5 * Math.pow(10, 9)),
+        gas: web3.utils.toHex(318730),
```

Comment:
do we want to pass the gas amount or let metamask ?

Reply:
looks like we pass it in new bounty form as well! So to keep it common -> I kept it the same

Reply:
hmm yeah was this just a relic of the gas needed for std bounties registration?